lib/github_changelog_generator/generator/generator_processor.rb

```diff
@@ -159,12 +159,14 @@ def include_issues_by_labels(issues)
     # @param [Array] issues Issues & PRs to filter when without labels
     # @return [Array] Issues & PRs without labels or empty array if
     #                 add_issues_wo_labels or add_pr_wo_labels are false
-    def filter_wo_labels(issues)
-      if (!issues.empty? && issues.first.key?("pull_requests") && options[:add_pr_wo_labels]) || options[:add_issues_wo_labels]
-        issues
-      else
-        issues.select { |issue| issue["labels"].map { |l| l["name"] }.any? }
+    def filter_wo_labels(items)
```

Comment:
I had to change the internals of this method beyond the initial issue of `pull_request` being incorrectly plural.

The original implementation, with the corrected value, would return all pull requests even when `:add_pr_wo_labels` was false when `:add_issues_wo_labels` was true. This was exposed when writing tests.

Reply:
A downside is that, when issues are passed in, the entire array will be iterated over to check for a `pull_request` key. I suppose that I could restore the `items.any? && items.first.key?("pull_request")` version if that is preferred